lib/ansible/modules/cloud/amazon/ec2_elb_lb.py

```diff
@@ -405,6 +405,11 @@
 except ImportError:
     HAS_BOTO = False
 
+try:
+    basestring
+except NameError:
+    basestring = str
```

Comment:
can you use `string_types` from `from ansible.module_utils.six import string_types` instead of just `str`